.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+[Add general information on the proposed Pull Request:
+describe benefits, changes, etc]
+
+
+## Related issue
+[Specify the issue this PR is meant to address]
+
+
+## Checklist
+[Make sure this PR provides:]
+- [ ] Docstring;
+- [ ] Example of usage;
+- [ ] Expanded example with reference links for website gallery;
+- [ ] Unittests.
+
+
+## Cross-references
+[Add links to related pages (discussions, meta-issues, etc)]
```

Comment:
## References
[If known, link to related Issues, PRs, or other online resources.]